templates/systemd-networkd-vxlan.netdev.j2

```diff
@@ -0,0 +1,10 @@
+[NetDev]
+Name={{ item.device }}
+Kind=vxlan
+
+[VXLAN]
+Id={{ item.vni | default(vxlan_vni) }}
+Group={{ item.group }}
+Local=0.0.0.0
```

Comment:
What's this for?

Reply:
I think this needs to use:
ipaddr: the IPV4 address assigned to the VXLAN interface (unsupported when using NetworkManger)

Reply:
I think this is the local address used to send  and receive VXLAN packets. Probably we don't need to set it.